src/test/SkipCallsTestHook.sol

```diff
@@ -199,6 +202,8 @@ contract SkipCallsTestHook is BaseTestHooks, Test {
         address payer = abi.decode(hookData, (address));
         int256 delta0 = IPoolManager(manager).currencyDelta(address(this), key.currency0);
         int256 delta1 = IPoolManager(manager).currencyDelta(address(this), key.currency1);
+        manager.sync(key.currency0);
+        manager.sync(key.currency1);
         IERC20Minimal(Currency.unwrap(key.currency0)).transferFrom(payer, address(manager), uint256(-delta0));
         IERC20Minimal(Currency.unwrap(key.currency1)).transferFrom(payer, address(manager), uint256(-delta1));
```

Comment:
technically this test is a bad practice bd if the `currency1` calls `settle(currency0)` in `transferFrom`, it would be able to take those tokens that have been sent in but not settled.
So maybe good practice just to change the order here to sync-send-settle-sync-send-settle

Reply:
good point - sauces lib will fix this and make sure people dont make the same mistake. Will change here though.